packages/app-desktop/gui/MainScreen/commands/newNote.ts

```diff
@@ -16,7 +15,7 @@ export const runtime = (): CommandRuntime => {
 			const folderId = Setting.value('activeFolderId');
 			if (!folderId) return;
 
-			const body = template ? TemplateUtils.render(template) : '';
+			const body = template ? template : '';
```

Comment:
Please remove anything related to templates from the code. That includes this variable and the signature of this function. Also make sure you update all code that calls "newNote", such as the "newTodo" command.

Reply:
Oh ok, missed it. Done.